keyboards/crkbd/keymaps/mcrown/mcrown.h

```diff
@@ -0,0 +1,137 @@
+/** @file mcrown.h
+ *  @brief mcrown header thant incluedes key layout and some definintions.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author Mario Corona (mariocc@comunidad.unam.mx) 2021
+ *
+ */
+
+#ifndef MCROWN_KEYMAP_H_LIB
+#define MCROWN_KEYMAP_H_LIB
```

Comment:
For include guards, we usually use: 
```suggestion
#pragma once
```